README.md

```diff
@@ -1,28 +1,167 @@
+ - Built in C++ for performance, with Python bindings for ease of use and rapid prototyping, with options for maximizing performance
  - Distributed computation with message transfers over RMDA using UCX
- - Dynamic reconfiguration to scale up and out at runtime​. Requires no changes to pipeline configuration
- - Unopinionated data model. Messages of any type can be used in the pipeline
+ - Dynamic reconfiguration to scale up and out at runtime​; requires no changes to pipeline configuration
+ - Unopinionated data model: messages of any type can be used in the pipeline
  - Built from the ground up with asynchronous computation for mitigation of I/O and GPU blocking
- - Automatically handles backpressure and reschedules computation as needed
- - Focused on rapid pipeline development, with options for maximizing performance
+ - Automatically handles backpressure (when the sender is operating faster than the receiver can keep up) and reschedules computation as needed
+
+<!-- omit in toc -->
+### Anatomy of a SRF Pipeline
+
+![SRF Pipeline](docs/imgs/srf_pipeline.jpg)
+
+<!-- omit in toc -->
+## Table of Contents
+- [Installation](#installation)
+  - [Prerequisites](#prerequisites)
+  - [Conda Installation](#conda-installation)
+    - [Python Bindings](#python-bindings)
+    - [C++ Bindings](#c-bindings)
+    - [Full SRF Library](#full-srf-library)
+  - [Source Installation](#source-installation)
+  - [Docker Installation](#docker-installation)
+- [Quickstart Guide](#quickstart-guide)
+- [Contributing](#contributing)
+  - [Third-Party code](#third-party-code)
+
+
+## Installation
+SRF includes both Python and C++ bindings and supports installation via [conda](https://docs.conda.io/en/latest/), Docker, or from source.]
+
+### Prerequisites
+
+- Pascal architecture (Compute capability 6.0) or better
+- NVIDIA driver `450.80.02` or higher
+- [conda or miniconda](https://conda.io/projects/conda/en/latest/user-guide/install/linux.html)
+- If using Docker:
+  - [Docker](https://docs.docker.com/get-docker/)
+  - [The NVIDIA container toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker)
+
+### Conda Installation
+Installing via Conda is the easiest method for getting the SRF components and supports both the Python and C++ bindings. To install the SRF conda package and build the C++ and hybrid components, follow the steps below. Please note, Conda performance can be greatly increased via [Mamba](https://github.com/mamba-org/mamba). To install Mamba in an existing Conda environment, simply run:
+
+```bash
+conda install -c conda-forge mamba
+```
+
+If you choose to use Mamba rather than Conda, simply replace `conda` with `mamba` in the instructions below.
+
+To avoid specifying the channels in the Conda commands below:
+```bash
+conda config --env --add channels conda-forge &&\
+conda config --env --add channels nvidia &&\
+conda config --env --add channels rapidsai
+```
+
```

Comment:
```suggestion
If you wish to opt-in to nightly builds, add the following channels:

```
conda config --env --add channels nvidia/label/dev &&\
conda config --env --add channels rapidsai-nightly
```
```

Reply:
I don't think suggestions like a to add code blocks.